lib/stream.js

```diff
@@ -15,8 +15,15 @@ var log = isABib ? bunyan.createLogger({name: 'sierra-bib-retriever'}) : bunyan.
 // send data to kinesis Stream
 exports.streamPoster = (records, streamName) => {
   return writeToStreamsClient(streamName, records)
+          .then(() => {
+            return Promise.resolve({sent: records.length, receieved: records.length})
```

Comment:
Thanks guys, I removed the check completely for sent vs received, because we are expecting it to not resolve until everything is sent successfully. If an error occurs, promise gets rejected and the function execution will fail completely.